boards/frdm-kw41z/include/board.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2017 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    board_frdm-kw41z Freescale FRDM-KW41Z Board
+ * @ingroup     boards
+ * @brief       Board specific implementations for the FRDM-KW41Z
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the FRDM-KW41Z
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * @brief   LED pin definitions and handlers
+ * @{
+ */
+#define LED0_PIN            GPIO_PIN(PORT_C,  1)
+
+#define LED0_MASK           (1 << 1)
+
+#define LED0_ON            (GPIOC->PCOR = LED0_MASK)
+#define LED0_OFF           (GPIOC->PSOR = LED0_MASK)
+#define LED0_TOGGLE        (GPIOC->PTOR = LED0_MASK)
```

Comment:
the board design is a bit conflicted, the pins used for the LEDs are also connected to the headers. I will post an updated version tomorrow